tests/docker-compose.yml

```
@@ -79,6 +82,14 @@ services:
     depends_on:
       - backend
 
+  modsec3-nginx-debug:
+    <<: *nginx
+    command: ["nginx-debug", "-g", "daemon-off;"]
```

Oops, I think that should be 
```suggestion
    command: ["nginx-debug", "-g", "daemon off;"]
```

---

I just went for the same entrypoint as the base one, but changed the command to nginx-debug, just in case.

---

Yes, perfect.